src/standard/x-styling.html

```diff
@@ -129,7 +131,7 @@
         // finally mixin properties inherent to this element
         this.mixin(props,
           propertyUtils.scopePropertiesFromStyles(this._styles));
-        this.mixin(props, this.customStyle);
+        propertyUtils.mixinCustomStyle(props, this.customStyle);

```

Comment:
Another good catch. I think a customStyle property should only apply if it's truthy.